lib/browser/api/menu.js

```diff
@@ -43,63 +43,32 @@ Menu.prototype._init = function () {
   }
 }
 
-Menu.prototype.popup = function (window, x, y, positioningItem) {
-  let [newX, newY, newPosition, newWindow] = [x, y, positioningItem, window]
-  let opts
-  let callback
-
-  // menu.popup(x, y, positioningItem)
-  if (window != null && !(window instanceof BrowserWindow)) {
-    [newPosition, newY, newX, newWindow] = [y, x, window, null]
-  }
+Menu.prototype.popup = function (options) {
+  let {window, x, y, positionItem, callback} = options
```

Comment:
I _think_ this breaks? The options argument is `positioningItem` but this line extracts `positionItem`

Reply:
ohh you're totally right